src/utils/hasRole.js

```diff
@@ -0,0 +1,10 @@
+const { userRepo } = require('../components/user/repo');
```

Comment:
Population of the userRepo action over the util for all other is maybe careful. It break the rule that the repo layer actions can not access. Maybe we should move this utils to users/utils/ and not to overall utils.
The same for hasSameSchool.